pyo3-ffi/src/object.rs

```diff
@@ -123,38 +131,45 @@ pub struct PyVarObject {
     pub ob_size: Py_ssize_t,
 }
 
-// skipped _PyVarObject_CAST
+// skipped private _PyVarObject_CAST
 
 #[inline]
 pub unsafe fn Py_Is(x: *mut PyObject, y: *mut PyObject) -> c_int {
     (x == y).into()
 }
 
+// skipped private _Py_GetThreadLocal_Addr
+
+// skipped private _Py_ThreadId
+
+// skipped private _Py_IsOwnedByCurrentThread
+
 #[inline]
-#[cfg(Py_GIL_DISABLED)]
 pub unsafe fn Py_REFCNT(ob: *mut PyObject) -> Py_ssize_t {
-    let local = (*ob).ob_ref_local.load(Relaxed);
-    if local == _Py_IMMORTAL_REFCNT_LOCAL {
-        return _Py_IMMORTAL_REFCNT;
+    #[cfg(Py_GIL_DISABLED)]
```

Comment:
I like this pattern of exposing the function once and then using per-config blocks inside the function to do the per-config implementation. Much cleaner than having multiple copies of the signature. And it looks like it plays nicer with the doc build too.